typescript/github-proxy/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "@hyperlane-xyz/github-proxy",
+  "description": "Github proxy that adds the API key to requests",
+  "version": "5.1.0",
+  "private": true,
```

Comment:
Is this the correct initial version? Will changeset bump this

Reply:
Yes it's the correct version but I'm not certain changeset will bump it given that it's private. But it's fine either way

Reply:
Removed private